packages/select/test/queryListTests.tsx

```diff
@@ -3,16 +3,15 @@
  *
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
-
 import { assert } from "chai";
 import { mount, ReactWrapper, shallow } from "enzyme";
 import * as React from "react";
 import * as sinon from "sinon";
 
+import { IFilm, renderFilm, TOP_100_FILMS } from "@blueprintjs/docs-app";
```

Comment:
oh dude revert this change! it should be the relative path below. this is failing the tests on CircleCI too. 

how did this get introduced?